packages/directory/tests/integration/components/dir-sidebar-test.js

```diff
@@ -1,58 +1,31 @@
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 
-moduleForComponent('dir-sidebar', 'Integration | Component | dir sidebar', {
-  integration: true,
-  beforeEach() {
-    this.render(hbs`{{dir-sidebar}}`);
-  }
-});
-
-test('it renders', function(assert) {
-  assert.expect(5);
-
-  assert.ok(this.$('.dir-sidebar').is(':visible'),
-    'The sidebar component is rendered');
-
-  assert.deepEqual(this.$('.dir-sidebar__group').toArray().map(el => el.textContent.trim()),
-    [ 'My Stuff', 'Canned Stuff' ],
-    'The sidebar component has the right groups');
-
-  assert.equal(this.$('.dir-sidebar__group--selected').text().trim(),
-    'My Stuff',
-    'The first group is initially selected');
-
-  assert.deepEqual(this.$('.dir-sidebar__filter').toArray().map(el => el.textContent.trim()),
-    [ 'All', 'Favorites', 'Recently Updated' ],
-    `The selected group's filters are shown in the filter section`);
-
-  assert.equal(this.$('.dir-sidebar__filter--selected').text().trim(),
-    'All',
-    'The first filter for the group is initially selected');
-});
-
-test('change selections', function(assert) {
-  assert.expect(4);
-
-  assert.equal(this.$('.dir-sidebar__group--selected').text().trim(),
-    'My Stuff',
-    'The first group is initially selected');
+module('Integration | Component | dir sidebar', function(hooks) {
+  setupRenderingTest(hooks);
 
-  this.$('.dir-sidebar__group:contains(Canned Stuff)').click();
+  hooks.beforeEach(async function() {
+    await render(hbs`{{dir-sidebar}}`);
+  });
 
-  assert.equal(this.$('.dir-sidebar__group--selected').text().trim(),
-    'Canned Stuff',
-    'The selected group is now `Canned Stuff`');
+  test('it renders', function(assert) {
+    assert.expect(4);
 
-  this.$('.dir-sidebar__group:contains(My Stuff)').click();
+    assert.ok(this.$('.dir-sidebar').is(':visible'),
```

Comment:
With the new testing style, we should use `this.element.querySelector` instead of `this.$`